home/templates/pages/about.html

```diff
@@ -30,6 +32,95 @@ <h1 class="display-2 mb-4">
     </div>
   </section>
   <!-- End of Hero section -->
+  <div class="whiteboard-container">
+    <h2>Kudoboard</h2>
+    <div id="whiteboard" class="whiteboard"></div>
+    <textarea id="reviewText" rows="3" placeholder="Type your review here..."></textarea>
+    <div>
+      <button id="submitReview">Submit</button>
+      <button id="clearReviews" disabled>Clear</button>
+    </div>
+  </div>
+  
+  <script>
+  document.addEventListener('DOMContentLoaded', function () {
+    const whiteboard = document.getElementById('whiteboard');
+    const reviewText = document.getElementById('reviewText');
+    const submitReview = document.getElementById('submitReview');
+    const clearReviews = document.getElementById('clearReviews');

```

Comment:
Kudoboard is working, but please update the submit and clear buttons to match the current website's style.